src/v2/Apps/Home/Components/HomeArtworkModuleContext.tsx

```diff
@@ -0,0 +1,249 @@
+import { Box, BaseTabs, BaseTab, Spacer, Text, Flex } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { HomeArtworkModuleContext_context } from "v2/__generated__/HomeArtworkModuleContext_context.graphql"
+
+interface HomeArtworkModuleContextProps {
```

Comment:
(non-blocking): Maybe just `HomeArtworkModules`?

Reply:
hm?

Reply:
Because `Context` is an overloaded term